pkg/dashboard/helpers_test.go

```diff
@@ -0,0 +1,239 @@
+package dashboard
+
+import (
+	"testing"
+
+	"github.com/fairwindsops/polaris/pkg/config"
+	"github.com/fairwindsops/polaris/pkg/validator"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWarningWidth(t *testing.T) {
+	input1 := validator.CountSummary{
+		Successes: 0,
+		Warnings:  0,
+		Dangers:   0,
+	}
+	input2 := 6
+
+	expectedOutput := uint(0x6)
+	actual := getWarningWidth(input1, input2)
+
+	assert.Equal(t, expectedOutput, actual)
+
+	input1 = validator.CountSummary{
+		Successes: 10,
+		Warnings:  3,
+		Dangers:   1,
+	}
+	input2 = 3
+
+	expectedOutput = uint(0x2)
+	actual = getWarningWidth(input1, input2)
+
+	assert.Equal(t, expectedOutput, actual)
+}
+func TestSuccessWidth(t *testing.T) {
+	input1 := validator.CountSummary{
+		Successes: 0,
+		Warnings:  0,
+		Dangers:   0,
+	}
+	input2 := 6
+
+	expectedOutput := uint(0x6)
+	actual := getSuccessWidth(input1, input2)
+
+	assert.Equal(t, expectedOutput, actual)
+
+	input1 = validator.CountSummary{
+		Successes: 8,
+		Warnings:  6,
+		Dangers:   4,
+	}
+	input2 = 7
+
+	expectedOutput = uint(0x3)
+	actual = getSuccessWidth(input1, input2)
+
+	assert.Equal(t, expectedOutput, actual)
+}
+func TestGetGrade(t *testing.T) {
+	input := validator.CountSummary{
+		Successes: 10,
+		Warnings:  3,
+		Dangers:   1,
+	}
+	expectedOutput := "B-"
+	actual := getGrade(input)
+
+	assert.Equal(t, expectedOutput, actual)
+	assert.NotEqual(t, "A+", actual)
+}
+
+func TestGetWeatherIcon(t *testing.T) {
+	input := validator.CountSummary{
+		Successes: 10,
+		Warnings:  3,
+		Dangers:   1,
+	}
+
+	expectedOutput := "fa-cloud-sun"
+	actual := getWeatherIcon(input)
+
+	assert.Equal(t, expectedOutput, actual)
+	assert.NotEqual(t, "fa-cloud-showers-heavy", actual)
+}
+
+func TestGetResultClass(t *testing.T) {
+	input := validator.ResultMessage{
+		ID:       "",
+		Message:  "",
+		Details:  []string(nil),
+		Success:  false,
+		Severity: "",
+		Category: "",
+	}
+
+	expectedOutput := " failure"
+	actual := getResultClass(input)
+
+	assert.Equal(t, expectedOutput, actual)
+	assert.NotEqual(t, " success", actual)
+
+	input = validator.ResultMessage{
+		ID:       "",
+		Message:  "",
+		Details:  []string(nil),
+		Success:  true,
+		Severity: "",
+		Category: "",
+	}
+
+	expectedOutput = " success"
+	actual = getResultClass(input)
+
+	assert.Equal(t, expectedOutput, actual)
+	assert.NotEqual(t, " failure", actual)
+}
+
+func TestGetWeatherText(t *testing.T) {
+	input := validator.CountSummary{
+		Successes: 10,
+		Warnings:  3,
+		Dangers:   1,
+	}
+
+	expectedOutput := "Mostly smooth sailing"
+	actual := getWeatherText(input)
+
+	assert.Equal(t, expectedOutput, actual)
+	assert.NotEqual(t, "Storms ahead, be careful", actual)
+}
+
+func TestGetIcon(t *testing.T) {
+	input := validator.ResultMessage{
+		ID:       "",
+		Message:  "",
+		Details:  []string(nil),
+		Success:  false,
+		Severity: "",
+		Category: "",
```

Comment:
You can omit all these fields, and just do `input := validator.ResultMessage{}`

In go, every type has a "zero value", i.e. `0` for integers, `0.0` for floats, `""` for strings, `false` for booleans, `[]` for slices, etc. When creating a struct, those values get used if you don't specify anthing.